RedHatInsights/curiosity-frontend
# build(containers): sw-1792 remove travis

## What's included

- build(containers): sw-1792 remove travis

### Notes
- additional task of removing travis from the repo settings required
- look at restoring the build passing badge as a subsequent task



## How to test
1. Confirm the GitHub actions linting and build...
   - jobs
      - steps pass
      - steps provide clear messaging







## Example

...

## Updates issue/story

sw-1792
